charts/rs-server-catalog/values.yaml

```diff
@@ -31,6 +31,11 @@ app:
   catalogBucket: rs-cluster-catalog
   # -- Presigned URL expiration time in seconds. 30 min by default
   presignedUrlExpirationTime: 1800
+  # values used to update the catalog stac metadata
+  metadata:
+    id: "rs-python"
```

Comment:
Can you use the comment with the `# --` tag like for the other parameters. Thanks to this, the description will appears in the documentation :)

Reply:
sure, I didn't realize this, thank you for pointing it out